x-pack/plugins/stack_alerts/server/rule_types/es_query/lib/fetch_search_source_query.ts

```diff
@@ -27,8 +33,14 @@ export async function fetchSearchSourceQuery(
 
   const initialSearchSource = await searchSourceClient.create(params.searchConfiguration);
 
+  const index = initialSearchSource.getField('index') as DataView;
+  if (!isTimeBasedDataView(index)) {
```

Comment:
Is there another issue to prevent rules from being created with non time based data views? Typically we like to prevent situations where users can create rules that error with every execution.

Reply:
Yes, there is another issue for this case #135806